requirements.txt

```diff
@@ -1,5 +0,0 @@
-numpy==1.9.1
-pandas==0.15.2
```

Comment:
@gradedSystem it looks like you're still using Pandas here https://github.com/datasets/nasdaq-listings/blob/main/scripts/process.py#L2

Reply:
I just moved it in the PR that you merged [here](https://github.com/datasets/nasdaq-listings/pull/1): 
Here is the file: https://github.com/gradedSystem/nasdaq-listings/blob/ef33e221022890eba6135821692910ec0c63b65e/scripts/requirements.txt